go/service/getdriveletter_windows.go

```diff
@@ -0,0 +1,98 @@
+// Copyright 2015 Keybase, Inc. All rights reserved. Use of
+// this source code is governed by the included BSD license.
+
+// +build windows
+
+package service
+
+import (
+	"strings"
+	"syscall"
+	"unsafe"
+)
+
+func getVolumeName(RootPathName string) (string, error) {
+	var VolumeNameBuffer = make([]uint16, syscall.MAX_PATH+1)
+	var nVolumeNameSize = uint32(len(VolumeNameBuffer))
+	var VolumeSerialNumber uint32
+	var MaximumComponentLength uint32
+	var FileSystemFlags uint32
+	var FileSystemNameBuffer = make([]uint16, 255)
+	var nFileSystemNameSize = syscall.MAX_PATH + 1
+
+	kernel32, _ := syscall.LoadLibrary("kernel32.dll")
+	getVolume, _ := syscall.GetProcAddress(kernel32, "GetVolumeInformationW")
```

Comment:
GetVolumeInformationW is fine. Probably a comment for the function that it needs a volume name that ends with a slash is warranted.